arcade-drive-xbox-controller/robot.py

```diff
@@ -1,14 +1,20 @@
 #!/usr/bin/env python3
-"""
-    This is a demo program showing the use of the DifferentialDrive class.
-    Runs the motors with split arcade steering and an Xbox controller.
-"""
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
 
 import wpilib
 import wpilib.drive
 
 
 class MyRobot(wpilib.TimedRobot):
+    """
+    This is a demo program showing the use of the DifferentialDrive class.
+    Runs the motors with split arcade steering and an Xbox controller.
+    """
```

Comment:
Personally I think these should remain as module docstrings?

Reply:
Agreed.  The idea behind this was consistency across all the examples.  Since it's at the class level in Java and some of the other python examples, and there's already a comment at the file/module level for the copyright, it seemed better to do it here.